README.md

```diff
@@ -1,22 +1,25 @@
 # ReactiveMongo for Play Framework
 
-This is a plugin for [Play Framework](https://www.playframework.com) 2.4 and 2.5, enabling support for [ReactiveMongo](http://reactivemongo.org) – a reactive, asynchronous and non-blocking Scala driver for MongoDB.
+This is a module for the [Play Framework](https://www.playframework.com) 2.5, 2.6 and 2.7, enabling support for [ReactiveMongo](http://reactivemongo.org) – a reactive, asynchronous and non-blocking Scala driver for MongoDB.
 
 ## Usage
 
 In your `project/Build.scala` or `build.sbt`:
```

Comment:
I would rather remove the section content and add a link to the documentation site, to avoid duplication.

Reply:
OK, sure. This link? http://reactivemongo.org/releases/0.1x/documentation/tutorial/play.html

Reply:
yes